### DISCUSSION TITLE
copilot can generate someone's api key (security vulnerability)

### DISCUSSION THREAD

**Discussion by 00dhkim** [community/community#6811] [Copilot Conversations]
When I code like this, copilot can return a specific api key.

I don't know it works or not, but I think it may be work and this could be a weakness(vulnerability).

please review this issue. thank you.

![image](https://user-images.githubusercontent.com/25413312/139058208-2de4f08a-30f9-4dc3-af29-91cfd9f9fd93.png)
![image](https://user-images.githubusercontent.com/25413312/139059097-bfb73256-a737-4eca-85a0-1aaa5109d1e9.png)

**Comment by hsenag** [comment]
Hi, thanks for letting us know about this. In general from our internal testing, it's extremely rare for Copilot to produce real API keys. Typically what it does is to produce made-up information synthesized from patterns in the training data.

For reference, we do have a [FAQ entry about this possibility](https://copilot.github.com/#faq-does-github-copilot-ever-output-personal-data).
